forge-game/src/main/java/forge/game/player/Player.java

```diff
@@ -455,7 +455,14 @@ public final boolean setLife(final int newLife, final SpellAbility sa) {
         boolean change = false;
         // rule 119.5
         if (life > newLife) {
-            change = loseLife(life - newLife, false, false) > 0;
+            final Map<Player, Integer> lossMap = Maps.newHashMap();
```

Comment:
Shouldn't this be in LifeSetEffect too?

Reply:
Yes

Reply:
Hmm, I thought I got that effect... I'll check

Reply:
Ah, I see what I did wrong now – thanks!

Reply:
While I was in there, I noticed that Reverse the Sands didn't really give you a choice in choosing players... so I tried to straighten that out as well.